community/runit/runit.initd

```diff
@@ -8,7 +8,10 @@ description="starts and monitors a collection of runsv(8) processes"
 retry="SIGHUP/5"
 
 depend() {
-    need localmount
-    after firewall
+	need localmount
+	after firewall
+	if [ -x /sbin/socklog ]; then
+		provide logger
+	fi
```

Comment:
I don’t understand this. `/sbin/socklog` is provided by the package socklog, not runit, and runit does not depend on socklog, so what does it do here?